README.md

```diff
@@ -103,6 +103,19 @@ repo configuration like:
         args: [-w]
         types: [shell]
 ```
+An alternative configuration for running shfmt via the official shfmt docker image is:
+```yaml
+  - repo: local
+    hooks:
+      - id: shfmt
+        name: Format Shell Script Files
+        description: Runs shfmt Docker image to format Shell Script files
```

Comment:
also for consistency with the other example, remove the description, as IMO it's a bit redundant